src/Compilers/CSharp/Test/Syntax/IncrementalParsing/IncrementalParsingTests.cs

```diff
@@ -2341,6 +2341,51 @@ public void InsertOpenBraceBeforeCodes()
             CompareIncToFullParseErrors(reparsedTree, parsedTree);
         }
 
+        [WorkItem(6676, "https://github.com/dotnet/roslyn/issues/6676")]
+        [Fact]
+        public void InsertExpressionStatementWithoutSemicolonBefore()
+        {
+            SourceText oldText = SourceText.From(@"
```

Comment:
>SourceText.From [](start = 33, length = 15)

This is probably doesn't parse the code as a Script. Also, should confirm what mode is used by  WithChangedText.